src/main/java/com/epam/ta/reportportal/core/project/impl/ProjectInfoWidgetDataConverter.java

```diff
@@ -265,7 +268,7 @@ public ProjectInfoWidgetDataConverter(@Qualifier("groupingStrategy") Map<InfoInt
 		Map<String, List<Launch>> result = new LinkedHashMap<>();
 		LocalDate prevDate = null;
 		for (Launch launch : initial) {
-			final LocalDate localDate = launch.getStartTime().toInstant(ZoneOffset.UTC).atZone(ZoneId.systemDefault()).toLocalDate();
+			final LocalDate localDate = launch.getStartTime().toInstant(ZoneOffset.UTC).atZone(ZoneId.of("UTC")).toLocalDate();
```

Comment:
maybe exists a more clear way to convert it

Reply:
LocalDateTime.now().atZone(ZoneOffset.UTC).toLocalDate();

Reply:
done